### ISSUE TITLE
slovakia vat number problem

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chadjlucas/vatnumber#9]
```
With this slovakian vat number, the check is not working:

SK2021853504

```

Original issue reported on code.google.com by `y...@pcsol.be` on 28 Jul 2010 at 12:49

**Comment by GoogleCodeExporter** [comment]
```
Fixed with changeset 0db4b0622e68
Thanks
```

Original comment by `cedric.krier@b2ck.com` on 28 Jul 2010 at 12:56
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Issue 12 has been merged into this issue.
```

Original comment by `cedric.krier@b2ck.com` on 19 Jan 2011 at 4:41
